packages/delivery-fetch/delivery.js

```diff
@@ -0,0 +1,44 @@
+import payload from '@bugsnag/core/lib/json-payload'
+
+const delivery = (client) => ({
+  sendEvent: (event, cb = () => {}) => {
+    const url = client._config.endpoints.notify
+
+    global.fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Bugsnag-Api-Key': client._config.apiKey,
+        'Bugsnag-Payload-Version': '4',
```

Comment:
Minor: Would it make sense to make this a constant somewhere so it can be shared between here and XHR (and any other places)?

Reply:
I was questioning this as well actually - sounds like a good idea to me.

Reply:
will raise a separate ticket to make this enhancement outside the scope of this PR